src/UI/Component/Input/Field/Select.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/* Copyright (c) 2018 Jesús López <lopez@leifos.com> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Input\Field;
+
+/**
+ * This describes select field.
+ */
+interface Select extends Input{
+
+	/**
+	 * Add a disabled first option "Choose one"
+	 *
+	 * @return Select
+	 */
+	public function withFirstOptionDisabled();
```

Comment:
How does this relate to the `required` property of the general input and the outcommented `withNoneOption` below? I somehow feel they are related, because: If a field is not required, I _need_ a "none of these"-option. We would not want to have a first disabled option saying "choose one" then. If I the field is required, but no value is currently set, I would always want to have a disabled "choose one" option. There seems to be some overlap.